.github/workflows/release.yml

```diff
@@ -171,14 +168,11 @@ jobs:
         with:
           go-version: 1.17
           check-latest: true
-      - name: "Get Tag From Environment"
-        id: get-tag
-        run: printf '::set-output name=tag::%s' "$(printf '%s' "${{ github.ref }}" | sed 's/refs\/tags\///')"
       - name: Retrieve Windows MSI Installer
         uses: actions/download-artifact@v2
         with:
-          name: observiq-otel-collector-${{ steps.get-tag.outputs.tag }}.msi
-          path: observiq-otel-collector-${{ steps.get-tag.outputs.tag }}.msi
+          name: observiq-otel-collector.msi
+          path: observiq-otel-collector.msi
```

Comment:
Forgot to mention this, but we'll also need to change the goreleaser yaml so that the glob includes this file.

Reply:
I double check I thought it was `observiq-otel-collector*.msi` so it would be covered.

Reply:
Ahh no it has a dash. Good catch!